### PR TITLE
[release-0.18] Bump xanzy/go-gitlab and go-playground/webhooks deps

### PR DESCRIPTION
**Pull request by antoineco**
Backport of #26 to release-0.18 branch.

### CODE REVIEW

File: pkg/adapter/receive_adapter.go [hunk 1]
```diff
@@ -222,7 +222,7 @@ func sourceFromGitLabEvent(gitLabEvent gitlab.Event, payload interface{}) string
 		return wpe.ObjectAttributes.URL
 	case gitlab.PipelineEvents:
 		pe := payload.(gitlab.PipelineEventPayload)
-		return pe.ObjectAttributes.URL
+		return pe.Project.HTTPURL
```
Comment (sebgoa) [inline pkg/adapter/receive_adapter.go:225]: is that going to be the same syntax then before ? e.g `https://gitlab.com/sebgoa/kdemo`

**Review by sebgoa — APPROVED** [review]
somehow I thought we were not vendoring deps anymore.

File: pkg/adapter/receive_adapter.go (hunk 1 above)
Comment (antoineco) [reply pkg/adapter/receive_adapter.go:225]: Yes, but this value isn't used anymore due to #24.

FYI this is a **backport**, there is no change to be made in this PR.

**Pull request merged**